expr/functions/integralByInterval/function.go

```diff
@@ -0,0 +1,108 @@
+package integralByInterval
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/bookingcom/carbonapi/expr/helper"
+	"github.com/bookingcom/carbonapi/expr/interfaces"
+	"github.com/bookingcom/carbonapi/expr/types"
+	"github.com/bookingcom/carbonapi/pkg/parser"
+	dataTypes "github.com/bookingcom/carbonapi/pkg/types"
+)
+
+type integralByInterval struct {
+	interfaces.FunctionBase
+}
+
+func GetOrder() interfaces.Order {
```

Comment:
Please, add a comment in the appropriate form.

Reply:
I don't understand the meaning of this function, I don't know how to add the comment.

Reply:
Well, you wrote it) Add a stub at least to make linter happy.

Reply:
um, ok